trio/_util.py

```diff
@@ -72,9 +72,11 @@ def signal_raise(signum):
 if sys.version_info < (3, 5, 2):
 
     def aiter_compat(aiter_impl):
-        @wraps(aiter_impl)
+        # de-sugar decorator to fix Python 3.8 coverage issue
+        # https://github.com/python-trio/trio/pull/784#issuecomment-446438407
         async def __aiter__(*args, **kwargs):
             return aiter_impl(*args, **kwargs)
+        __aiter__ = wraps(aiter_impl)
```

Comment:
```suggestion
        __aiter__ = wraps(aiter_impl)(__aiter__)
```

Reply:
Thanks! I just did that. I'll worry about dropping 3.5 in another pull request.